createdisk.sh

```diff
@@ -166,36 +182,8 @@ fi
 # This must be done after the generation of libvirt image as it reuses some of
 # the content of $libvirtDestDir
 if [ "${SNC_GENERATE_MACOS_BUNDLE}" != "0" ]; then
-    start_vm ${VM_NAME} ${VM_IP}
-    if [ ${BUNDLE_TYPE} != "okd" ]; then
-        # workaround https://github.com/crc-org/vfkit/issues/11 on macOS 12
-        downgrade_rhel9_kernel ${VM_IP}
-        cleanup_vm_image ${VM_NAME} ${VM_IP}
-    fi
-
-    # Get the rhcos ostree Hash ID
-    ostree_hash=$(${SSH} core@${VM_IP} -- "cat /proc/cmdline | grep -oP \"(?<=${BASE_OS}-).*(?=/vmlinuz)\"")
-
-    # Get the rhcos kernel release
-    kernel_release=$(${SSH} core@${VM_IP} -- 'uname -r')
-
-    # Get the kernel command line arguments
-    kernel_cmd_line=$(${SSH} core@${VM_IP} -- 'cat /proc/cmdline')
-
-    # Get the vmlinux/initramfs to /tmp/kernel and change permission for initramfs
-    ${SSH} core@${VM_IP} -- "mkdir /tmp/kernel && sudo cp -r /boot/ostree/${BASE_OS}-${ostree_hash}/*${kernel_release}* /tmp/kernel && sudo chmod 644 /tmp/kernel/initramfs*"
-
-    # SCP the vmlinuz/initramfs from VM to Host in provided folder.
-    ${SCP} -r core@${VM_IP}:/tmp/kernel/* $INSTALL_DIR
-
-    ${SSH} core@${VM_IP} -- "sudo rm -fr /tmp/kernel"
-    shutdown_vm ${VM_NAME}
-
     vfkitDestDir="${destDirPrefix}_vfkit_${destDirSuffix}"
     rm -fr ${vfkitDestDir} ${vfkitDestDir}.crcbundle
-
-    create_qemu_image "$libvirtDestDir"
```

Comment:
? shoudn't this happen?

Reply:
nope it is already done as part of libvirt bundle creation.